inst/tests/tests.Rraw

```diff
@@ -13593,7 +13593,7 @@ test(1962.087, dcast(DT, a ~ b, value.var = 'b'),
 test(1962.088, dcast(DT[0L, ], a ~ c, value.var = 'b'), data.table(a=numeric(), key="a")) #1215
 test(1962.089, dcast(DT, a ~ c, value.var = 'b'),
      data.table(a = c(1, 2), `2` = c(0L, 2L), `4` = c(2L, 0L), key = 'a'),
-     message = 'Aggregate function missing')
+     message = "fun.aggregate is NULL, but found duplicate row/column combinations, so defaulting to length(). That is, the variables [a, c] used in 'formula' do not uniquely identify rows in the input 'data'. In such cases, 'fun.aggregate' is used to derive a single representative value for each combination in the output data.table, for example by summing or averaging (fun.aggregate=sum or fun.aggregate=mean, respectively). See ?dcast.data.table for more details.")
```

Comment:
[a, c] is helpful, good

Reply:
I would not test against the full contents of the message -- that will require us to change the test for minor issues like small copyediting/grammar/wording changes that don't materially affect the message.

Better would be a regex that captures the most important part of the message, e.g.

"fun.aggregate is NULL.*defaulting to length\\(\\).*\\[a, c\\]"

Reply:
> I would not test against the full contents of the message -- that will require us to change the test for minor issues like small copyediting/grammar/wording changes that don't materially affect the message.
> 
> Better would be a regex that captures the most important part of the message, e.g.
> 
> "fun.aggregate is NULL._defaulting to length\(\)._\[a, c\]"

done 👍